hope/presskit.html

```diff
@@ -28,20 +27,17 @@ <h3>Information</h3>
   <h3>Gameplay</h3>
 
   <p>
-    You play as Agatha Hope, a leader of The Cicadas, a rebel movement fighting against the tyranny of a mega-corporation
-    called NexxusCorp.
+    As the titular character and Captain, Agatha Hope, it's your responsibility to unite a group of selfinterested individuals, to explore an unfamiliar spaceship and survive.
   </p>
 
   <p>
-    Your mission is to infiltrate NexxusCorp's lunar complex, seize Dr. Mariko Nishimura and steal the
-    prototype starship Dragon X13. Upon capturing the ship you will set course to Proxima B, where you will establish the
-    first free human settlement beyond the Solar System and, more importantly, beyond NexxusCorp's reach.
+    To do that, you first will need a ship. And lucky for you, tyranic NexxusCorp has a prototype ready for the taking. 
+    Your mission is to infiltrate NexxusCorp's lunar complex, <s>kidnap</s> seize Dr. Mariko Nishimura and <s>steal</s> acquire the
```

Comment:
Not sure I would play with HTML tags here to format text (strikethrough). I imagine people might copy this and put it in places where HTML is not even supported.